nixos/modules/services/system/dbus.nix

```diff
@@ -147,6 +147,9 @@ in
       };
 
       systemd.services.dbus = {
+        aliases = [
+          "dbus-broker.service"
```

Comment:
This is for the dbus implementation, not the dbus-broker implementation. Please add here (and further down) a comment stating this is a hack due to our activation script misbehaving when switching back and forth.

Reply:
Its actually in both cases helping to prevent the issue but for dbus-broker is has a real reason, that other units can just depend on dbus.